tastyworks/models/option_chain.py

```diff
@@ -64,24 +86,39 @@ async def get_option_chain(session, underlying: Underlying, expiration: date = N
             strike_val = Decimal(strike['strike-price'])
             for option_types in OptionType:
                 new_option = Option(
+                    symbol='',
                     ticker=underlying.ticker,
-                    expiry=exp_date,
+                    expiration_date=exp_date,
+                    dte=int(exp['days-to-expiration']),
                     strike=strike_val,
                     option_type=option_types,
-                    underlying_type=UnderlyingType.EQUITY
+                    underlying_type=UnderlyingType.EQUITY,
+                    shares_per_contract=data['shares-per-contract'],
+                    expiration_type=exp['expiration-type'],
+                    settlement_type=exp['settlement-type'],
+                    greeks=Greeks
                 )
                 res.append(new_option)
     return OptionChain(res)
 
 
 async def _get_tasty_option_chain_data(session, underlying) -> Dict:
+    """
+    https://api.tastyworks.com/option-chains/SPY/nested
+    Args:
+        session:
+        underlying:
+
+    Returns:
+
```

Comment:
Do you mind adding descriptions to this docstring stating what the arguments & return value mean?

Reply:
Will make sure to do that moving forward. I see in some other methods that there is the type of the arguments added in parenthesis before the colon. What is the recommended format overall?